web/src/components/filings-calendar/tax-access-modal/TaxAccessStepTwo.test.tsx

```diff
@@ -188,6 +188,38 @@ describe("<TaxAccessStepTwo />", () => {
       });
     });
 
+    it("updates businessName on submit if tax filing is success", async () => {
```

Comment:
[pebble]can we create another version of this test for responsibleOwner name?

Reply:
Initially added it, but realized it's already being tested on line 433 `"updates taxId and responsibleOwnerName on submit"`